Source/relay/Guide/Items of the Month/Better Shrooms and Gardens.ash

```diff
@@ -11,7 +11,7 @@ void IOTMBetterShroomsAndGardensGenerateResource(ChecklistEntry [int] resource_e
         int freeFightsLeft = totalFreeFights - freeFightsUsed;
 
         // Title
-        string main_title = freeFightsLeft + " free fight";
+        string main_title = pluralise(freeFightsLeft,"free fight","free fights");
```

Comment:
Can we move this to the "Free Fights" tile? Maybe we want to keep the extra tile if they are in plumber?